PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -83,13 +105,19 @@ def __init__(self, cfg_ana, cfg_comp, looperName):
           if   doResidual == "MC":   doResidual = self.cfg_comp.isMC
           elif doResidual == "Data": doResidual = not self.cfg_comp.isMC
           elif doResidual not in [True,False]: raise RuntimeError, "If specified, applyL2L3Residual must be any of { True, False, 'MC', 'Data'(default)}"
-          GT = getattr(cfg_comp, 'jecGT', mcGT if self.cfg_comp.isMC else dataGT)
+          GTs = getattr(cfg_comp, 'jecGT', mcGT if self.cfg_comp.isMC else dataGT)
```

Comment:
here for backwards compatibility you need something like
`if type(GTs) == str: GTs = [ (0, GT) ]`